wiki/FAQ/en.md

```diff
@@ -1,731 +1,99 @@
-[Formula given here][Score#Level wikilink].
-
-#### Why is my profile stats different ingame?
-Keep in mind that _osu!_ has four game modes so you might be looking at the wrong game mode.
-
-![Profile_Mode_Buttons][Profile_Mode_Buttons image]
-
-_Manually select the game mode to view at the profile._
-
-#### How do I change the default keyboard key input for my game mode of choice?
-_Full explanation: [Options][Options wikilink]_
-
-![Options sidebar with quickicons][Options quickicons image]
-
-From the _Input_ icon in the _Options_ sidebar as shown above, under **Keyboard** section as shown below.
-
-![Options keyboard section][Options keyboard image]
-
-#### Game over? Where did I mess up?
-_Full explanation: [Replay][Replay wikilink]_
-
-After failing a beatmap, press the `F1` key to replay the currently failed play.
-The replay ends when the screen darkens, with an osu!logo in the middle, rather than the game over screen.
-To rewatch the replay again, hold the custom default grave/tilde (`` `/~ ``) key (can be changed in Options sidebar; inside _Change keyboard bindings_ as **Quick Retry (hold briefly)**) or `Ctrl`+`R` for a while.
-
-Due to the non-standard replay screen, use the `Esc` key to return to Song Selection.
-Retrying the play (as in, actual play) of the beatmap in this state is not possible.
-It currently works in Solo mode only when given the game over screen.
-**This replay will not be saved nor allowed to be exported.**
-To record the failed replay, please use external third-party software for now since _osu!_ has yet to have native export support for failed replay.
-
-#### Does osu! has keyboard shortcuts?
-_Full list: [Shortcut Key Reference][Shortcut Key Reference wikilink]_
-
-Pressing the `Insert` key (in default keyboard setting) will hide, pause (if playing) and mute _osu!_ into the _Windows_ sidebar.
-Find the _osu!logo_ in the _Windows_ sidebar, and click on it to return _osu!_ to the state before pressing the key.
-It is called the **Boss key**, and can be modified in `Change keyboard bindings` (in `Options`, under _Keyboard_ section), located at the end of the _Universal_ section.
-
-Another interesting shortcut is `Alt`+`Enter`.
-The shortcut toggles the fullscreen mode button in `Options`, but does not work during gameplay or in Multi.
-
-#### What should I backup when fresh installing osu! on a new device?
-The whole `osu!` folder, including hidden files, is the short answer.
-On the new device after running `osu!install.exe` and completing _osu!_ installation successfully, copy the backup `osu!` folder and merge to the new device's `osu!` folder.
-If it ask for merging `.dll` or `.exe` files, ignore it.
-Allow merging for everything else, including `.cfg` and `.db` files.
 
-#### Can I use osu! on my mobile device?
-_Full list: [Game Modes under External Ports section][Game Modes#External Ports wikilink]_
+### What are those "Performance Points"?
```

Comment:
hmm, you should just remove this question and replace it with `_Main page: [Performance Points](/wiki/Performance_Points)_` because Performance Points should already have explained what they are